src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String REGEX = " ";
```

Comment:
REGEX is a bad name for the variable, rename it 
```suggestion
    public static final String SPACE_REGEX = " ";
```
OR
```suggestion
    public static final String SPACE_DELIMITER = " ";
```